guides.md

```diff
@@ -237,6 +239,140 @@ members. Skip the steps below at your discretion for low-stakes topics.
   1. Use the private dial-in number from our [technical Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
   2. _After_ prompt, enter conference code `307 314 3734 #`
 
+## Onboarding Checklist
+
+This is a checklist for onboarding a new Member to the Co-operative 🚀
+
+### Setting up communications
+
+1. Get Signal number and add to group.
+
+1. Get Matrix account and invite to spaces:
+  - Private chat `#hyphacoop-private:tomesh.net`
+  - Public chat `#hyphacoop-open:tomesh.net`
+  - Community `+hyphacoop:tomesh.net`
+
+1. Set up [`hypha.coop` email](https://link.hypha.coop/email):
+  - Create new mailbox with Mailcow
+  - Set up initial forwarder
+  - Add to `members@`
+
+### Setting up virtual office
+
+1. Invite to [Passbolt](https://pass.hypha.coop):
+  - After acceptance, add to `Member-Worker` group
+
+1. Invite to [GitHub Organization](https://github.com/hyphacoop):
+  - Add to `Member-Worker` [GitHub Team](https://link.hypha.coop/teams)
+
+1. Get Google-friendly email address:
+  - Share access to [Drive](https://link.hypha.coop/drive)
+  - Invite to calendars:
+    - https://link.hypha.coop/calendar
+    - https://link.hypha.coop/availability
+
+1. Invite to [Loomio](https://loomio.hypha.coop/).
+
+1. Share [weekly schedule](https://link.hypha.coop/schedules) (recommend without password).
+
+### Setting up employee record and payroll
+
+1. Collect information for [employee record](https://link.hypha.coop/employees).
+
+1. Add Member SIN to Passbolt and share access with `People Operations` group.
+
+1. Collect encrypted (see [Sensitive Data](#sensitive-data) guide) [`TD1` and `TD1-ON` forms](https://www.canada.ca/en/revenue-agency/services/forms-publications/td1-personal-tax-credits-returns.html) to [Drive](https://link.hypha.coop/drive). 
```

Comment:
Maybe just collect TD1 and then encrypt it ourselves? Less odd work for new person. The normal process is just a forgettable "here sign this paper", so would be neat to stay close to the ease of that.

Reply:
This doesn't help because how do we get a piece of important information electronically in the first place? We also want ppl to learn how to encrypt and transfer files.